google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseClient.java

```diff
@@ -239,4 +239,6 @@
    *
    */
   TransactionRunner readWriteTransaction();
+  
+  TransactionManager transactionManager();
```

Comment:
We need a doc comment here, which would warn users that they probably don't need this for 99% of cases and should use readWriteTransaction() instead.